vue-components/src/components/Dropdown.vue

```diff
@@ -253,8 +254,15 @@ $base: '.wikit-Dropdown';
 		position: absolute;
 	}
 
+	&__label-wrapper {
+		display: flex;
+		align-items: center;
+		gap: $dimension-layout-xxsmall;
+	}
+
 	&__label {
 		@include Label( block );
+		display: inine;
```

Comment:
typo: this should probably be `inline`? Similarly in the other files.

However, **this line can just be removed**, it is obviously not needed as it is currently ignored and the every looks fine.

Reply:
ah.. ofc this line is useless. thanks